check_copyright.py

```diff
@@ -32,8 +34,6 @@
 from thefuzz import fuzz
 
 IDF_PATH = os.getenv('IDF_PATH', os.getcwd())
```

Comment:
I think this should be removed as well if want this tool to be used for other projects as well.